Mono.Addins.Gui/Mono.Addins.Gui/NewSiteDialog.cs

```diff
@@ -29,14 +29,34 @@
 
 using System;
 using Gtk;
+using Mono.Addins.Setup;
+using Mono.Unix;
 
 namespace Mono.Addins.Gui
 {
 	partial class NewSiteDialog : Dialog
 	{
+		ComboBox typeComboBox;
 		public NewSiteDialog (Gtk.Window parent)
 		{
 			Build ();
+			var hbox = new HBox ();
+			hbox.Spacing = 6;
+			hbox.PackStart (new Label(), false, false, 10);
+			var label = new Label ();
+			label.Text = Catalog.GetString ("Type:");
+			hbox.PackStart (label, false, false, 1);
+			typeComboBox = new ComboBox (new string [] {
+				Catalog.GetString ("Add-in Repository"),//AddinRepositoryType.MonoAddins
+				Catalog.GetString ("Visual Studio Marketplace")//AddinRepositoryType.VisualStudioMarketplace
```

Comment:
I don't think we should allow end users to add "Marketplace" type repos. That is a repo kind we now support, but custom repositories should use the standard type.

Reply:
Reason I added this so QA can test on private instance of marketplace, should I make it changeable via enviorment variable for QA?

Reply:
yes, we can use an env var for this, like we do for the updater